lib/render/leaderboard.php

```diff
@@ -245,18 +245,72 @@ function RenderScoreLeaderboardComponent($user, $friendsOnly, $numToFetch = 10)
     echo "</div>";
 }
 
-function RenderTopAchieversComponent($gameTopAchievers)
+/**
+ * Creates the High scores tables on game pages
+ * 
+ * @param String $user The logged in user.
+ * @param Array $gameTopAchievers Top 10 highest scoreres for the game.
+ * @param Array $gameLatestMasters Top 10 latest masters for the game.
```

Comment:
Great to see meaningful DocBlocks here 👍  A minor thing: please use lowercase variants to make it clear those are native types, otherwise it refers to non-existing classes (Array, String -> array, string).